pbft/pbft.hpp

```diff
@@ -208,7 +209,7 @@ namespace bzn
         bool move_to_new_configuration(const hash_t& config_hash, uint64_t view);
         bool proposed_config_is_acceptable(std::shared_ptr<pbft_configuration> config);
 
-        void maybe_record_request(const pbft_msg& msg, const std::shared_ptr<pbft_operation>& op);
+        void maybe_record_request(const pbft_msg& msg, const bzn_envelope& env, const std::shared_ptr<pbft_operation>& op);
```

Comment:
Does this really need to take both a msg and envelope?

Reply:
I will review maybe_record_request and see about cleaning it up.

Reply:
Reviewed and rewritten with Isabel